babi.py

```diff
@@ -1411,6 +1413,10 @@ def sanitanize(x):
                         vals.append(sanitanize(babi_eval(x, record,
                             convert_none='zero')))
                     except Exception as message:
+                        with  Transaction() as new_transaction:
+                            self.traceback = x + repr(message)
+                            self.save()
+                        notify('Error on expression: %s' % x, priority=1)
```

Comment:
Given that the notify message is for the user, maybe it's better to just notify the babi.report name.

And add the expression to the traceback.